Sources/AppStoreConnectCLI/Commands/TestFlight/BetaTesters/CreateBetaTesterCommand.swift

```diff
@@ -84,7 +84,7 @@ struct CreateBetaTesterCommand: CommonParsableCommand {
             .map(\.data)
             .sink(
                 receiveCompletion: Renderers.CompletionRenderer().render,
-                receiveValue: Renderers.ResultRenderer(format: common.outputFormat).render
+                receiveValue: { _ in }
```

Comment:
Following what we have for other Commands (`CancelUserInvitationsCommand` `DeleteBetaTesterCommand` etc), no error or no result means success.